subprojects/platform-play/src/main/java/org/gradle/play/internal/twirl/PlayTwirlAdapterV23X.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.play.internal.twirl;
+
+import org.gradle.language.twirl.TwirlImports;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+class PlayTwirlAdapterV23X implements VersionedPlayTwirlAdapter {
+
+    private static final Iterable<String> SHARED_PACKAGES = Arrays.asList("play.twirl.compiler", "scala.io"); //scala.io is for Codec which is a parameter to twirl
+
+    // Based on https://github.com/playframework/playframework/blob/2.4.0/framework/src/build-link/src/main/java/play/TemplateImports.java
+    private static final List<String> DEFAULT_JAVA_IMPORTS = Collections.unmodifiableList(Arrays.asList(
```

Comment:
The `COMMON_IMPORTS` could be used here too (and in other adapters).

Reply:
Same response as above